docs/Overview.md

```diff
@@ -0,0 +1,52 @@
+# ForgeGradle
+
+Welcome to ForgeGradle, Minecraft Forge's Gradle plugin. It is a small, simple,
+and effective plugin with the primary purpose of bootstrapping Minecraft Forge's
+toolchain to aid in the development of mods targeting Minecraft Forge.
+
+## Basic Usage
+
+### Applying the Plugin
+
+ForgeGradle 7 can be applied to projects or settings by using the following:
+
+```groovy
+plugins {
+    id 'net.minecraftforge.gradle' version '7.0.0'
```

Comment:
Replacing 7.0.0 with <version> or similar would help prevent people from adding an old version, instead encouraging them to look on Gradle Plugin Portal. See EventBus' readme for an example of this.